src/app.jsx

```diff
@@ -459,6 +464,170 @@ class Application extends React.Component {
+                    copyQuadletContainers[id] = container;
+            });
+
+            for (const key of Object.keys(quadlets.containers)) {
+                const quadlet = quadlets.containers[key];
+                const container_key = makeKey(con.uid, key);
+
+                // Mock podman container state
+                const container = {
+                    uid: con.uid,
+                    key: container_key,
+                    Id: key,
+                    IsService: false,
+                    IsInfra: false,
+                    // unique quadlet identifier
+                    IsQuadlet: true,
+                    Name: quadlet.name,
+                    ImageName: quadlet.image,
+                    NetworkSettings: {
+                        Ports: [],
+                    },
+                    Mounts: [],
+                    Config: {
+                        // Cmd: quadlet.exec.split(' ')
+                        Labels: {
+                            PODMAN_SYSTEMD_UNIT: key,
+                        },
+                    },
+                    // HACK, hardcoded, needs mapping from systemd and O(n) calls
+                    State: {
+                        Status: 'exited'
+                    }
+                };
+
+                const found_pod = podNameServiceMap[quadlet.pod];
+                if (found_pod) {
+                    container.Pod = found_pod;
+                }
+                copyQuadletContainers[container.key] = container;
+            }
+
+            const users = prevState.users.map(u => u.uid === con.uid ? { ...u, quadletsLoaded: true } : u);
+            return { quadletContainers: copyQuadletContainers, quadletPods: copyQuadletPods, users };
+        });
+    }
+
+    async subscribeDaemonReload(con) {
+        // We don't support subscribing on reload events for "other" users.
+        if (con.uid !== 0 && con.uid !== null) {
+            return;
+        }
+
+        console.log('subscribe daemon reload', con);
+
+        let options = { };
+        if (con.uid === 0) {
+            options = { bus: "system", superuser: "try" };
+        } else {
+            options = { bus: "session" };
+        }
+
+        const subscribe = (client) => {
+            const subscription = client.subscribe({ interface: "org.freedesktop.systemd1.Manager", member: "Reloading" }, (_path, _iface, _signal, [reloading]) => {
+                if (!reloading)
+                    this.initQuadlets(con);
+            });
+
+            this.setState(prevState => {
+                const users = prevState.users.map(u => u.uid === con.uid ? { ...u, dbus: { client, subscription } } : u);
+                return { users };
+            });
+        };
+
+        let client = null;
+        const user = this.state.users.find(u => u.uid === con.uid);
+
+        // don't add multiple Reload subscriptions
+        if (user?.dbus?.subscription) {
+            return;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2266-fa7f0a3b-20250922-124438-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)